### PR TITLE
#133 Changed design

### PR DESCRIPTION
**Pull request by Chopikov**
- Changed xml
- Changed background image
- Changed buttons style

### CODE REVIEW

File: app/src/main/res/layout-land/activity_start.xml [hunk 1]
```diff
@@ -27,19 +27,30 @@
 
         <Button
             android:id="@+id/button_load_image"
-            style="@style/AppTheme.Button.Start"
-            android:text="@string/action_load_image" />
+            android:layout_width="wrap_content"
```
Comment (maik-nack) [inline app/src/main/res/layout-land/activity_start.xml]: Please move these lines in style `AppTheme.Button.Start`

File: app/src/main/res/layout/activity_start.xml [hunk 2]
```diff
@@ -27,19 +27,30 @@
 
         <Button
             android:id="@+id/button_load_image"
-            style="@style/AppTheme.Button.Start"
-            android:text="@string/action_load_image" />
+            android:layout_width="wrap_content"
```
Comment (maik-nack) [inline app/src/main/res/layout/activity_start.xml]: Please move these lines in style `AppTheme.Button.Start`

File: app/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -28,15 +28,14 @@
 
     <style name="AppTheme.Button.Start"
         parent="Widget.AppCompat.Button.Colored">
-        <item name="colorButtonNormal">@color/colorRiverBed</item>
         <item name="android:layout_width">wrap_content</item>
         <item name="android:layout_height">wrap_content</item>
-        <item name="android:layout_marginTop">0dp</item>
+        android:layout_height="wrap_content"
```
Comment (maik-nack) [inline app/src/main/res/values/styles.xml]: What is it?

**Review by maik-nack — APPROVED** [review]
GOOD JOB!!!

**Review by MaximSmolskiy — APPROVED** [review]
Looks great!

**Pull request merged**